interfaces/IOCDM.h

```diff
@@ -0,0 +1,384 @@
+ /*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+#pragma once
+
+
+#include "Module.h"
+
+
+namespace WPEFramework {
+namespace Exchange {
+
+
+typedef enum : uint32_t {
+    OCDM_SUCCESS = 0,
+    OCDM_S_FALSE = 1,
+    OCDM_KEYSYSTEM_NOT_SUPPORTED = 0x80000002,
+    OCDM_INVALID_SESSION = 0x80000003,
+    OCDM_INVALID_DECRYPT_BUFFER = 0x80000004,
+    OCDM_OUT_OF_MEMORY = 0x80000005,
+    OCDM_FAIL = 0x80004005,
+    OCDM_INVALID_ARG = 0x80070057,
+    OCDM_SERVER_INTERNAL_ERROR = 0x8004C600,
+    OCDM_SERVER_INVALID_MESSAGE = 0x8004C601,
+    OCDM_SERVER_SERVICE_SPECIFIC = 0x8004C604,
+} OCDM_RESULT;
+
+// ISession defines the interface towards a DRM context that can decrypt data
+// using a given key.
+struct ISession : virtual public WPEFramework::Core::IUnknown {
```

Comment:
and again I guess it was already in the original file but now we are touching it anyway let's remove WPEFramework. Please check all occurrences in this file